Content.Server/Administration/Systems/AdminVerbSystem.Smites.cs

```diff
@@ -551,23 +552,26 @@ private void AddSmiteVerbs(GetVerbsEvent<Verb> args)
             };
             args.Verbs.Add(killSign);
 
-            // TODO: Port cluwne outfit.
             Verb clown = new()
             {
-                Text = "Clown",
+                Text = "Cluwne",
                 Category = VerbCategory.Smite,
-                IconTexture = "/Textures/Objects/Fun/bikehorn.rsi/icon.png",
+                IconTexture = "/Textures/Clothing/Mask/cluwne.rsi/icon.png",
                 Act = () =>
                 {
-                    SetOutfitCommand.SetOutfit(args.Target, "ClownGear", EntityManager, (_, clothing) =>
+                    var meta = MetaData(args.Target);
+                    _popupSystem.PopupEntity(Loc.GetString("cluwne-transform", ("target", args.Target)), args.Target, PopupType.LargeCaution);
+                    meta.EntityName = Loc.GetString("cluwne-name-prefix", ("target", meta.EntityName));
```

Comment:
use proxy method Name() (also test that this concats the name correctly with spaces and all that)

Reply:
Thanks AJCM this should be fixed although i went ahead and gave the cluwne an accent and scream so you might want to just recheck those new elements.